exercises/simple-linked-list/Example.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+public class SimpleLinkedList<T>
+{
+    public SimpleLinkedList(T value) :
+        this(value, null)
+    {
+    }
+
+    public SimpleLinkedList(T value, SimpleLinkedList<T> next)
+    {
+        Value = value;
+        Next = next;
+    }
+
+    public T Value { get; }
+
+    public SimpleLinkedList<T> Next { get; }
+
+    public static SimpleLinkedList<T> FromEnumerable(IEnumerable<T> values)
```

Comment:
Ah, I should have looked at the exercise more closely. Well shoot, now I question if implementing `ICollection` would complicate things rather than help. Yeah, let's forget about `Head`, `Push`, `Pop` & `Peek`. You can leave the class as-is as well. Sorry about the misunderstanding.